health/micro-ui-react19/web/packages/css/src/pages/employee/commodityManagement.scss

```
@@ -0,0 +1,492 @@
+  color: #0B4B66;
+  margin: 0;
+}
+
+.cm-search-input {
+  padding: 0.5rem 0.75rem;
+  border: 0.063rem solid #D6D5D4;
+  border-radius: 0.25rem;
+  font-size: 0.875rem;
+  width: 12.5rem;
+  outline: none;
+  font-family: "Roboto", sans-serif;
```

Stylelint (17.6.0)

[error] Unexpected quotes around "Roboto" (font-family-name-quotes)

(font-family-name-quotes)